dev/tasks/macros.jinja

```diff
@@ -204,7 +204,7 @@ on:
     env:
       ARROW_FORMULA: ./arrow/dev/tasks/homebrew-formulae/{{ formula }}
     run: |
-      brew update
+      brew update || :
```

Comment:
Do we maybe want to echo that there was an issue with brew update?

Reply:
you mean something like: `brew update || echo "brew update did not finish successfully"`?